docs/src/componentDocs/ChannelValue/examples/ChannelValueWithDifferentUnits.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import * as colors from '@brightlayer-ui/colors';
+import { ChannelValueWithDifferentUnitsExample } from './ChannelValueWithDifferentUnitsExample';
+
+const codeSnippet = `<Box sx={{ display: 'flex', flexDirection: 'column', '& .BluiChannelValue-root': { my: 0.25 } }}>
+    <ChannelValue value="85" units="kWh" />
```

Comment:
```suggestion
    <ChannelValue value="85" units="kWh" />
    <ChannelValue value="85" units="kWh" unitSpace={'show'} />
```

Reply:
Added examples using unitSpace hide and show